Formula/a/autobrr.rb

```diff
@@ -0,0 +1,46 @@
+class Autobrr < Formula
+  desc "Modern, easy to use download automation for torrents and usenet"
+  homepage "https://autobrr.com/"
+  url "https://github.com/autobrr/autobrr/archive/refs/tags/v1.36.0.tar.gz"
+  sha256 "631ee5f858c5952e1d7a8c27b13fe33aea9021ffd814bb540f0e494291168309"
+  license "GPL-2.0-or-later"
+
+  depends_on "go" => :build
+  depends_on "node" => :build
+
+  def install
+    system "npx", "pnpm", "install", "--dir", "web"
+    system "npx", "pnpm", "--dir", "web", "run", "build"
+
+    ldflags = "-s -w -X main.version=#{version} -X main.commit=#{tap.user}"
+
+    system "go", "build", *std_go_args(output: bin/"autobrr", ldflags: ldflags), "./cmd/autobrr"
+    system "go", "build", *std_go_args(output: bin/"autobrrctl", ldflags: ldflags), "./cmd/autobrrctl"
+  end
+
+  def post_install
+    (var/"autobrr").mkpath
+  end
+
+  service do
+    run [opt_bin/"autobrr", "--config", var/"autobrr/"]
+    keep_alive true
+    log_path var/"log/autobrr.log"
+  end
+
+  test do
+    pid = fork do
+      exec "#{bin}/autobrr", "--config", testpath
```

Comment:
Is there a way to specify a port?

Reply:
Only from within the config file, which will be created on first launch.

Reply:
Can you manually create a config file to pass in as part of the test?

Reply:
Done! Thanks for the patience and feedback on this ❤️

Reply:
@s0up4200 Thank you for your patience with us and the back and forth. Writing your first formula can be a little confusing, but we really appreciate your contribution.